src/components/TokenSelect.js

```diff
@@ -31,6 +31,9 @@ const TokenSelect = (props) => {
 
     const balance = item.uid in props.tokensBalance
       ? props.tokensBalance[item.uid].available : 0;
+
+    const isNFT = item.uid in props.tokenMetadata && props.tokenMetadata[item.uid].nft;
```

Comment:
Same comment from my `hathor-wallet` PR, add a helper `isNFT` method somewhere and re-use it everywhere

Reply:
I agree. Did this refactor, can you se again please?